### DISCUSSION TITLE
bin/test failed on a fresh clone.

### DISCUSSION THREAD

**Discussion by harshanabandara** [sympy/sympy#25178] [Q&A]
I want to contribute to sympy but I'm getting 5 test failures and DO NOT COMMIT message when I run bin/test. Can someone hep me out?

**Comment by oscarbenjamin** [comment]
Can you show the test failure output?

**Reply by harshanabandara** [reply]
![image](https://github.com/sympy/sympy/assets/58684412/63e82b43-f8d9-4958-80d1-9acb4ce03349)
![image](https://github.com/sympy/sympy/assets/58684412/447e178c-f8d5-4687-be87-82abde8a423d)
![image](https://github.com/sympy/sympy/assets/58684412/281b799b-b1bf-45c2-8874-c232be70a2e8)
The NotImplementedError is a error I raise from a function. You can ignore it.

**Reply by oscarbenjamin** [reply]
> The NotImplementedError is a error I raise from a function. You can ignore it.

This implies that it is not a fresh clone and that you have made some changes. Try again with a fresh clone.

**Reply by harshanabandara** [reply]
![Screenshot from 2023-05-27 11-49-03](https://github.com/sympy/sympy/assets/58684412/686fc4d1-8166-4e8c-90b8-16915f5e9792)
This is the output from a fresh clone.
